### PR TITLE
Preview image: prevent NPE

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
- prevent NPE: this was reported on google crash console:
```
java.lang.ArrayIndexOutOfBoundsException: 
  at java.util.Vector.get (Vector.java:748)
  at com.owncloud.android.ui.preview.PreviewImagePagerAdapter.getFileAt (PreviewImagePagerAdapter.java:135)
  at com.owncloud.android.ui.preview.PreviewImageActivity.onPageSelected (PreviewImageActivity.java:354)
```
- code enhancement

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/preview/PreviewImagePagerAdapter.java [hunk 1]
```diff
@@ -207,187 +199,29 @@ public int getItemPosition(Object object) {
...
-        return mDownloadErrors.contains(Integer.valueOf(position));
-    }
-
     /**
      * Reset the image zoom to default value for each CachedFragments
      */
-    public void resetZoom() {
-        Iterator<FileFragment> entries = mCachedFragments.values().iterator();
-        while (entries.hasNext()) {
-        FileFragment fileFragment = entries.next();
```
[the remark this answers is not in the document]
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/preview/PreviewImagePagerAdapter.java]: So AndroidStudio says
> Access can be package-private, thus removing "public"

and Codacy says we should use explicit scoping.

**Comment by mario** [comment]
@tobiasKaminsky please rebase this.

**Comment by tobiasKaminsky** [comment]
@mario done, ready for review :-)

**Comment by mario** [comment]
👍

**Comment by tobiasKaminsky** [comment]
Merging with only one approval as @AndyScherzinger is on vacation.

**Pull request merged**